Android.mk

```diff
@@ -9,8 +9,6 @@ LOCAL_SRC_FILES := \
 LOCAL_MODULE := dirtycow
 LOCAL_LDFLAGS   += -llog
 LOCAL_CFLAGS    += -DDEBUG
-LOCAL_CFLAGS    += -fPIE
```

Comment:
`ndk-build` will dynamically set the `-fPIE` flag when it's supported (I have tested it with the newest NDK version). I have documented this in the [corresponding commit message](https://github.com/timwr/CVE-2016-5195/pull/46/commits/631787d4174d4c7e53b8d9879212a2fd53e0c7ee).